pkg/virtctl/vnc/vnc.go

```diff
@@ -84,37 +84,51 @@ type VNC struct {
 	clientConfig clientcmd.ClientConfig
 }
 
-func (o *VNC) Run(cmd *cobra.Command, args []string) error {
-	namespace, _, err := o.clientConfig.Namespace()
+func (o *VNC) setupLocalProxy(vmiName string) (*net.TCPListener, error) {
+	lnAddr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("127.0.0.1:%d", customPort))
 	if err != nil {
-		return err
+		return nil, fmt.Errorf("Can't resolve the address: %s", err.Error())
+	}
+	// The local tcp server is used to proxy the podExec websock connection to vnc client
```

Comment:
I believe that this comment explains the purpose of the function, it should probably be placed above the signature?
Maybe you can rename the function so the comment is not needed at all?

Reply:
@dankenigsberg You are correct. Applied more informative function name.